### PR TITLE
Removing run_pl_glue.py from text classification docs, include run_xnli.py & run_tf_text_classification.py

### PR DESCRIPTION
**Pull request by cbjuan**
# What does this PR do?

Since `run_pl_glue.py` is not part of `text-classification` examples after #9010, this PR removes it from the text-classification docs. Also, it adds `run_xnli.py` and `run_tf_text_classification.py` scripts, which are in that folder now.

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@sgugger may be interested in the PR, he's responsible for docs and the author of #9010

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Good catch! To make each version of the documentation point to the examples that go with it, the links should also be like this:
```
:prefix-link:`run_glue <examples/text-classification/run_glue.py>`__
```
This way the master will be replaced by the tag of the release when in the docs of a specific version.
```

**Comment by cbjuan** [comment]
Sure! Change applied

**Review by sgugger — APPROVED** [review]
Thanks!

One last thing, could you run `make style` on your branch so that the formatting is applied?

**Comment by cbjuan** [comment]
Done, command `make style` applied. Thanks for the guidance

File: docs/source/task_summary.rst [hunk 1]
```diff
@@ -54,12 +54,11 @@ Sequence Classification
...
 sequence classification is the GLUE dataset, which is entirely based on that task. If you would like to fine-tune a
-model on a GLUE sequence classification task, you may leverage the `run_glue.py
-<https://github.com/huggingface/transformers/tree/master/examples/text-classification/run_glue.py>`__ and
-`run_pl_glue.py
-<https://github.com/huggingface/transformers/tree/master/examples/text-classification/run_pl_glue.py>`__ or
-`run_tf_glue.py
-<https://github.com/huggingface/transformers/tree/master/examples/text-classification/run_tf_glue.py>`__ scripts.
+model on a GLUE sequence classification task, you may leverage the :prefix_link:`run_glue.py
+<examples/text-classification/run_glue.py>`__, :prefix_link:`run_tf_glue.py
+<examples/text-classification/run_tf_glue.py>`__, :prefix_link:` run_tf_text_classification.py
```
Comment (sgugger) [inline docs/source/task_summary.rst:59]:
```suggestion
<examples/text-classification/run_tf_glue.py>`__, :prefix_link:`run_tf_text_classification.py
```
I think this is what is making the doc building unhappy.

**Comment by cbjuan** [comment]
Build still not succeeding. I will check

**Comment by sgugger** [comment]
Ah yes, those links don't need the underscores! Good catch and sorry for giving you the wrong example to follow. Just waiting for the last tests and we can merge :-)

**Comment by cbjuan** [comment]
No worries, @sgugger. Thanks for the guidance :)

**Pull request merged**